agent/test-agent-cli/src/main.rs

```diff
@@ -0,0 +1,91 @@
+mod client;
+mod error;
+mod get_secret;
+mod init;
+mod retry_count;
+mod send_results;
+mod terminate;
+mod test_error;
+
+use crate::client::Client;
+use argh::FromArgs;
+use env_logger::Builder;
+use error::Result;
+use log::LevelFilter;
+use model::constants::ENV_TEST_NAME;
+use snafu::ResultExt;
+
+#[derive(FromArgs)]
+///This command line interface is to interact with K8s cluster when creating testcase using Bash script
+struct Args {
+    /// set logging verbosity [trace|debug|info|warn|error]. If the environment variable `RUST_LOG`
+    /// is present, it overrides the default logging behavior. See https://docs.rs/env_logger/latest
+    #[argh(option, default = "LevelFilter::Info")]
+    log_level: LevelFilter,
+
+    #[argh(subcommand)]
+    command: Command,
+}
+
+#[derive(FromArgs, PartialEq, Debug)]
+#[argh(subcommand)]
+enum Command {
+    ///Send any error occured
```

Comment:
nit: keep comment spacing consistent. Here and elsewhere
```suggestion
    /// Send any error encountered
```